server/controllers/article.js

```diff
@@ -259,3 +259,48 @@ export const deleteArticle = async (req, res) => {
     return serverError(res, error);
   }
 };
+
+export const rateArticle = async (req, res) => {
+  const { slug, user, article } = req;
+  //const article = await Articles.findOne({ where: { slug } });
+  const articleId = article.get("id");
+  const raterId = user.userId;
+  const { score } = req.body;
+  const validScore = score < 6 && score > 0;
+  try {
+    const isRating = await Ratings.findOne({
```

Comment:
using is before an object name indicates that this is a boolean value. please rename this object

Reply:
It is renamed to rating